test/screenshot/webpack-bundles.js

```diff
@@ -0,0 +1,63 @@
+module.exports.sassBundle = function(sassInput, cssOutput) {
```

Comment:
I think cutting this down from sassBundle and javaScript bundle into just 1 xyzBundle method will cut down the compile time. When you pass a new config into the array in multicompiler mode, webpack thinks it is a new bundle. Yes we want the css file to be separate, but I think by passing the `name` prop to the the `file-loader` will create a separate css file. 

The only drawback(up for debate) to having 1 bundle is that you must import sass files into the JS files. But I think this improves build times.